code/script.js

```diff
@@ -29,19 +59,643 @@ const showMessage = (message, sender) => {
+  
+    yesBtn.addEventListener('click', () => {
+      showMessage(`Yes!`, 'user')
+      ending()
+    })
+
+    yesBtn.onclick = function () {
+      if (yesBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+    noBtn.onclick = function () {
+      if (noBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+
+    noBtn.addEventListener('click', () => {
+      input.value=''
+      setTimeout (() => showMessage(`Too bad! I will make you start over`, 'bot'),1000);
+      setTimeout(() => beginAgain(), 5000);
+    })
+  }
+  else if (event.target.value === 'Carey') {
+    inputWrapper.innerHTML = `
+      <audio autoplay> 
+        <source src="./assets/Carey.mp3" type="audio/mp3">
+      </audio>
+  `
+  inputWrapper.innerHTML += `
+    <button type="submit" id="yes">Yes</button>
+    <button type="submit" id="no">No</button>
+    `
+
+    const yesBtn = document.getElementById('yes')
+    const noBtn = document.getElementById('no')
+  
+    yesBtn.addEventListener('click', () => {
+      showMessage(`Yes!`, 'user')
+      ending()
+    })
+
+    yesBtn.onclick = function () {
+      if (yesBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+    noBtn.onclick = function () {
+      if (noBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+
+    noBtn.addEventListener('click', () => {
+      input.value=''
+      setTimeout (() => showMessage(`Too bad! I will make you start over`, 'bot'),1000);
+      setTimeout(() => beginAgain(), 5000);
+    })
+  }
+ })
+
+  } else { //if the bot does not recognize the mood
+    showMessage(`I don't recognize that mood..`,'bot')
+    //dropdown with all the moods
+    inputWrapper.innerHTML = `
+    <select id="select5">
+     <option value="" selected disabled> Select a mood..</option>
+     <option value="happy">happy</option>
+     <option value="glad">glad</option>
+     <option value="excited">excited</option>
+     <option value="angry">angry</option>
+     <option value="mad">mad</option>
+     <option value="frustrated">fustrated</option>
+     <option value="irritated">irritated</option>
+     <option value="sad">sad</option>
+     <option value="depressed">depressed</option>
+     <option value="blue">blue</option>
+     <option value="in-love">in love</option>
+     <option value="love">love</option>
+     <option value="romantic">romantic</option>
+   </select>
+   <button id="rerun"> Go </button>
+   `
+//human can chose a mood in the dropdown and press go to listen
+   document.getElementById('rerun').onclick = function() {
+     console.log('now I have clicked', document.getElementById('select5').value);
+     songSelection(document.getElementById('select5').value)
+   }
+  
+  }
+
```

Comment:
might be the longest else if i've seen so far, but it is so smart and it includes a lot! Really cool how much information can be put in the statement and also nice to see how it works!